src/api/player/client/hasWeapon.lua

```diff
@@ -0,0 +1,11 @@
+---hasWeapon
+---@param weaponName string
+---@return bool
+---@public
+function API_Player:hasWeapon(weaponName)
+    if HasPedGotWeapon(PlayerPedId(), weaponName, false) == false then

```

Comment:
Has**Ped**GotWeapon suggest to move this functions from API_Player to API_Ped i think ?

Reply:
I have store here because i'have include PlayerPedId() in function. In API_Ped function there is pedId. I have test in API_Ped, but don't work, so i have not make in the PR ^^